test/merge.js

```diff
@@ -0,0 +1,72 @@
+import expect from "expect";
+import { parse, write, merge } from "webidl2";
+
+// Collapse sequences of whitespace to a single space.
+function collapse(s) {
+  return s.trim().replace(/\s+/g, " ");
+}
+
+expect.extend({
+  toMergeAs(received, expected) {
```

Comment:
Wow, didn't know `expect` supports extension this way 👍

Reply:
Me neither, I found out after I had repeated myself a lot and wanted a helper function :)